webapp/src/main/java/com/box/l10n/mojito/entity/PluralForm.java

```diff
@@ -20,6 +22,7 @@
             @Index(name = "UK__PLURAL_FORM__NAME", columnList = "name", unique = true)
         }
 )
+@BatchSize(size = 1000)
```

Comment:
There is no point to a higher number in this case - I didn't realise that the table will always have 6 types of plural forms (makes sense), I'll change that in the next iteration. I based that number on the row size and response time of similar tables for a similar number of rows.

(As far as I could tell debugging, Hibernate doesn't include duplicate IDs when doing multiple batches).

For locale, I initially chose 1000 as the batch size. The only reason I bumped that up is that if table expands with over 100 entries (new locales - maybe unlikely? - or fake/testing locales) - it would result in multiple batches even though we'd be better off getting them in the same batch (and not sure anyone would remember to check, increase and test the batch size after the fact).
Let me know what batch size would make most sense here.

Ideally, both the locale and plural_form would be cached at the Hibernate layer - so we only very rarely get SQL queries for them as they seem fairly static. That would only be a tiny perf improvement though, so probably not worth doing now. (Locales seems to be cached at the application - but not Hibernate layer).